.env.development

```diff
@@ -7,7 +9,10 @@ VITE_KC_SITE_BASE_URL=https://dev.zoo.dev
 VITE_KC_SITE_APP_URL=https://app.dev.zoo.dev
 VITE_KC_SKIP_AUTH=false
 VITE_KC_CONNECTION_TIMEOUT_MS=5000
-# ONLY add your token in .env.development.local if you want to skip auth, otherwise this token takes precedence!
-#VITE_KC_DEV_TOKEN="your token from dev.zoo.dev should go in .env.development.local"
+#VITE_KC_DEV_TOKEN="optional token from dev.zoo.dev to skip auth in the app"
+
+RUST_BACKTRACE=1
```

Comment:
I actually couldn't find docs on it, but it's essentially verbosity level. If you hit a Rust panic without it set, the CLI error message literally tells you to set it to `full`.

https://github.com/rust-lang/cargo/issues/1634#issuecomment-1448731075